feature/home/src/main/java/com/yapp/feature/home/component/HomeHeader.kt

```diff
@@ -192,7 +194,12 @@ private fun SessionItem(
             horizontalArrangement = Arrangement.spacedBy(8.dp),
             verticalAlignment = Alignment.CenterVertically
         ) {
-            SessionChip(progressPhase = progressPhase)
+            when (progressPhase) {
+                SessionProgressPhase.PENDING -> {
+                    if (relativeDays < 0) SessionChip(progressPhase = progressPhase)
+                }
+                else -> SessionChip(progressPhase = progressPhase)
+            }
```

Comment:
C: 해당 로직을 state 로 녹이는거에 대해선 어떻게 생각하시나요 ??

```suggestion
data class HomeSession(
    val id: String,
    val name: String,
    val place: String,
    val date: String,
    val dayOfWeek: String,
     val relativeDays: Int,
     val startTime: String?,
     val endTime: String?,
     private val progressPhase: SessionProgressPhase,
     val attendanceStatus: AttendanceStatus?
 ) {
    val showSessionChip: Boolean
    get() {
       if (progressPhase != SessionProgressPhase.PENDING) {
          true
       } else {
           if (relativeDays < 0) true
           else false
        }
      }
```

이렇게 녹이면 좋겠다고 생각한건 조건이 추가될 때마다 state 클래스 컴포넌트 양쪽 다 관리해야하는 포인트가
늘어나는거 같아서 제안했습니다 !

Reply:
넵 좋은 거 같아요!